src/Microsoft.SqlTools.ServiceLayer/SqlContext/QueryExecutionSettings.cs

```diff
@@ -163,6 +163,11 @@ public class QueryExecutionSettings : GeneralRequestDetails
         /// </summary>
         private const int DefaultQueryGovernorCostLimit = 0;
 
+        /// <summary>
+        /// Flag to run query in sqlcmd mode
```

Comment:
>Flag to run query in sqlcmd mode [](start = 12, length = 32)

Specifically this is the default for the flag

Reply:
good catch :)